package.json

```diff
@@ -63,6 +63,7 @@
   },
   "dependencies": {
     "prop-types": "^15.5.10",
+    "react-is": "^16.5.2",
```

Comment:
Can you move this to peerDeps?

Reply:
Sure. But it would be a breaking change, wouldn't it?